Makefile

```diff
@@ -0,0 +1,117 @@
+.PHONY: all
+all: build
+
+help: ## this help
```

Comment:
This target prints for me only `help    this help`?

Reply:
Fixed, should work now.